src/com/xilinx/rapidwright/util/function/InputStreamSupplier.java

```diff
@@ -49,7 +52,7 @@ static InputStreamSupplier fromPath(Path p) {
     public static InputStream getInputStream(Path fileName) {
         InputStream in = null;
         try {
-            if (fileName.toString().endsWith(".gz")) {
+            if (fileName.toString().endsWith(".gz") && Params.RW_DECOMPRESS_GZIPPED_EDIF) {
```

Comment:
`InputStreamSupplier` seems to be a generic utility that could be used outside of the EDIF parser, so maybe `decompressToDisk` should be an method parameter (which would also address the comment too)

Reply:
Updated the comment and added the new parameter.